File: swift day3- 语法高级/代码/17-OCCallSwift/OCCallSwift/ViewController.m
//
//  ViewController.m
//  OCCallSwift
//
//  Created by 1 on 16/6/17.
//  Copyright © 2016年 小码哥. All rights reserved.
//

#import "ViewController.h"
#import "OCCallSwift-Swift.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    Person *p = [[Person alloc] init];
    p.name = @"why";
    [p test];
}

@end

File: swift day3- 语法高级/代码/16-Swift调用OC代码/16-Swift调用OC代码/ViewController.swift
//
//  ViewController.swift
//  16-Swift调用OC代码
//
//  Created by 1 on 16/6/17.
//  Copyright © 2016年 小码哥. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let p = Person()
        p.name = "why"
        p.age = 18
        
        p.test()
    }
    
}

